### PR TITLE
[github] Split Chinese translation teams into zh-Hans and zh-Hant

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -81,7 +81,7 @@ android/src/fdroid/play/listings/mr-IN/ @organicmaps/translations-mr
 android/src/fdroid/play/listings/pt-BR/ @organicmaps/translations-pt
 android/src/fdroid/play/listings/ru-RU/ @organicmaps/translations-ru
 android/src/fdroid/play/listings/tr-TR/ @organicmaps/translations-tr
-android/src/fdroid/play/listings/zh-CN/ @organicmaps/translations-zh
+android/src/fdroid/play/listings/zh-CN/ @organicmaps/translations-zh-Hans
```
Comment (biodranik) [inline .github/CODEOWNERS:84]: Sholdn't it be listings/zh/ for simplified Chinese?

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -90,4 +90,4 @@ iphone/metadata/id/ @organicmaps/translations-id
 iphone/metadata/it/ @organicmaps/translations-it
 iphone/metadata/pt-BR/ @organicmaps/translations-pt
 iphone/metadata/tr/ @organicmaps/translations-tr
-iphone/metadata/zh-Hans/ @organicmaps/translations-zh
+iphone/metadata/zh-Hans/ @organicmaps/translations-zh-Hans
```
Comment (biodranik) [inline .github/CODEOWNERS:93]: Let's hard-code zh-Hant here too.

File: .github/CODEOWNERS (hunk 1 above)
Comment (biodranik) [reply .github/CODEOWNERS:84]: What should be other values for traditional one? HK, MO, TW? Should we also hard-code them?

**Review by biodranik — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:84]: > Sholdn't it be listings/zh/ for simplified Chinese?

Please don't start this flame war. Let's follow existing conventions in Android/iOS. Simplified (Mainland) will be zh-Hans, Traditional (HK, Macao, Taiwan) - zh-Hant. No discrimination. No decision we need to make on our own.

> What should be other values for traditional one? HK, MO, TW? Should we also hard-code them?

I am not a big expert in Chinese, but as far as I remember, Hong Kong, Macao, and Taiwan use **traditional** Chinese (zh-Hant), whereas Mainland, Singapore and few other places use Simplified. There are can be some local differences, like between British and American English variations. The difference between Simplified and Traditional is **huge**.

@organicmaps/translations-zh-Hans, @organicmaps/translations-zh-hant, please correct me if I am wrong.

**Comment by LaoshuBaby** [comment]
I'd prefer the `zh-Hans` + `zh-Hant` because looks like we won't add another kind of regional varient of Chinese in a long future, and we haven't got so many contributor can work on 3 or more team.

And the difference between zh-HK and zh-TW is smaller than difference between Simplified and Traditional. For zh-HK and zh-MO you even can see them same thing. In other project (out of OSM world), HK's local translator even ask for fallback to zh-TW: https://github.com/roundcube/roundcubemail/issues/6666

Besides, not only Mainland of China is using Simplified Chinese. **So if we only have 1 translator in zh-Hans team, 2 in zh-Hant, I think simply split to 2 team is enough.**

**Review by biodranik — APPROVED** [review]
Let's not forget about updating this file when adding new languages.

File: .github/CODEOWNERS (hunk 2 above)
Comment (rtsisyk) [reply .github/CODEOWNERS:95]: Added.

**Comment by groverlynn** [comment]
> I'd prefer the `zh-Hans` + `zh-Hant` because looks like we won't add another kind of regional varient of Chinese in a long future, and we haven't got so many contributor can work on 3 or more team.
> 
> And the difference between zh-HK and zh-TW is smaller than difference between Simplified and Traditional. For zh-HK and zh-MO you even can see them same thing. In other project (out of OSM world), HK's local translator even ask for fallback to zh-TW: [roundcube/roundcubemail#6666](https://github.com/roundcube/roundcubemail/issues/6666)
> 
> Besides, not only Mainland of China is using Simplified Chinese. **So if we only have 1 translator in zh-Hans team, 2 in zh-Hant, I think simply split to 2 team is enough.**
> 
> In some project there only contributor from Simplified using regions or Traditional using regions, so developer in there will simply use `zh` refer to Chinese, because they don't need to distinguish. For example, UN change to Simplified version because political reason, or you only can see Traditional Chinese on Spotify, Netflix because they don't need the mainland market (Aha, blocked).
> 
> **But OSM is world wide and both language varient have many contributor and user, so I think neither S or T can be seen as `zh` in current situation.**

`zh-TW` and `zh-CN` are (by default) both Mandarin. That's the same standard language with slightly different accents and no communication barriers. `zh-SG` and `zh-MY` would have a bit more Southeast Asian vocabulary (mostly Malay and Malay-English). `zh-HK` and `zh-MO` are (by default) cantonese-influenced/based written Chinese. That is a compromise between two different but related languages. Cantonese and Mandarin are orally incommunicable with each other, and written form can also be incommunicable if too much informal vocabulary is used. 
Whereas zh-Hant and zh-Hans refer to the scripts but not the language. It is analogical to `sr-Cyrl` (Serbo-Croatian in Cyrillic script) and `sr-Latn`(Serbo-Croatian in Latin script). Transforming between `-Hant` and `-Hans` scripts is called **transliteration**, but transforming between `yue` (Cantonese, a.k.a. Yue Chinese) and `cmn` (Mandarin Chinese) is called **translation**.
Notably, Apple uses `zh-HK` as the written interface language for `yue-Hant` (Cantonese in traditional Chinese script). Also, there are people using `zh-Hans-HK` and `zh-Hans-MO`, while `zh-Hant-MY` is actually the consensus for the news titles in Malaysia's Chinese newspapers.

**Comment by LaoshuBaby** [comment]
> `zh-TW` and `zh-CN` are (by default) both Mandarin. That's the same standard language with slightly different accents and no communication barriers. `zh-SG` and `zh-MY` would have a bit more Southeast Asian vocabulary (mostly Malay and Malay-English). `zh-HK` and `zh-MO` are (by default) cantonese-influenced/based written Chinese. That is a compromise between two different but related languages. Cantonese and Mandarin are orally incommunicable with each other, and written form can also be incommunicable if too much informal vocabulary is used.
> Whereas `zh-Hant` and `zh-Hans` differ in written scripts but not the language (both are macro-Chinese, covering all eight/ten major Chinese languages). It is analogical to `sr-Cyrl` (Serbo-Croatian in Cyrillic script) and `sr-Latn`(Serbo-Croatian in Latin script). Transforming between `-Hant` and `-Hans` scripts is called **transliteration**, but transforming between `yue` (Cantonese, a.k.a. Yue Chinese) and `cmn` (Mandarin Chinese) is called **translation**.
> Notably, Apple uses `zh-HK` as the written interface language for `yue-Hant` (Cantonese in traditional Chinese script). Also, there are people using `zh-Hans-HK` and `zh-Hans-MO`, while `zh-Hant-MY` is actually the consensus for the news titles in Malaysia's Chinese newspapers.

Clearly, the **written form**, not the **phonetic**, is being discussed here. And obviously the current energy of the community is appropriate to maintain two sets of scripts (**yes, this is not precise, but generally enough**)

**Comment by biodranik** [comment]
Thank you very much for the informational discussion! We learned a lot :)
While most of the texts in OM are related to the *written* form of Chinese, TTS strings (voice directions) are now wrongly using the written form instead of the *phonetic* one! Can you please elaborate on the relevant issue https://github.com/organicmaps/organicmaps/issues/1519 and propose the best solution for OM, while keeping existing TTS engines and their supported languages in mind?

**Pull request merged**